src/pcre2_jit_neon_inc.h

```diff
@@ -87,7 +87,7 @@ POSSIBILITY OF SUCH DAMAGE.
 	&& ((__clang_major__ == 3 && __clang_minor__ >= 3) || (__clang_major__ > 3)))
 __attribute__((no_sanitize_address))
 #endif
-static sljit_u8* SLJIT_FUNC FF_FUN(sljit_u8 *str_end, sljit_u8 **str_ptr, sljit_uw offs1, sljit_uw offs2, sljit_uw chars)
+static sljit_u8* SLJIT_FUNC FF_FUN(sljit_u8 *str_end, sljit_u8 **str_ptr, sljit_uw offs1, sljit_uw offs2, sljit_u32 chars)
```

Comment:
this should break the 16bit library

Reply:
Not AFAICT, though I must admit I'd initially focused the 8-bit one.